bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/component.tsx

```
@@ -516,104 +524,212 @@ const ChatMessageList: React.FC<ChatListProps> = ({
-                setShowStartSentinel(userScrolledUp);
-              }
-            }}
-            onWheel={(e) => {
-              if (e.deltaY < 0 && isStartSentinelVisible && !lockLoadingNewPages) {
-                setUserLoadedBackUntilPage((prev) => {
-                  if (typeof prev === 'number' && prev > 0) {
-                    setLockLoadingNewPages(true);
-                    return prev - 1;
-                  }
-                  return prev;
+          <Content>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)